nxdrive/gui/view.py

```diff
@@ -366,9 +367,10 @@ def __init__(self, translate: Callable, parent: QObject = None) -> None:
             self.UPLOADED: b"uploaded",
             self.TOTAL: b"total",
             self.ENGINE: b"engine",
-            self.CREATED_AT: b"created_at",
-            self.COMPLETED_AT: b"completed_at",
+            self.CREATED_ON: b"created_on",
+            self.COMPLETED_ON: b"completed_on",
```

Comment:
You have to update `Session` attributes also to keep the code coherent and easy reading/searching the whole code base.